### PR TITLE
Email verification and notifications bar

### PR DESCRIPTION
**Pull request by nathanstitt**
Moves component and polling from Tutor and sets up for reading account
email verification
Depends on PRs:
- Accounts Rails 201 fix: https://github.com/openstax/accounts-rails/pull/50
- Openstax API fixes to work with Rails 3.2: https://github.com/openstax/openstax_api/pull/44
- Accounts upgrade API for CORS: https://github.com/openstax/accounts/pull/310
- Tutor representer: https://github.com/openstax/tutor-server/pull/1071

screen shot 2016-05-05 at 5 33 22 pm

### CODE REVIEW

**Comment by nathanstitt** [comment]
This is ready for review and can be merged, but it won't actually work until the BE tutor and accounts PR's are merged.

File: index.cjsx [hunk 1]
```diff
@@ -1,51 +1,28 @@
...
-  ExerciseIdentifierLink,
-  KeysHelper
+  PinnedHeader,
+  PinnedHeaderFooterCard: require './src/components/pinned-header-footer-card'
+  Question:               require './src/components/question'
+  RefreshButton:          require './src/components/buttons/refresh-button'
+  ResizeListenerMixin:    require './src/components/resize-listener-mixin'
+  SmartOverflow:          require './src/components/smart-overflow'
+  SpyMode:                require './src/components/spy-mode'
 }
```
Comment (nathanstitt) [inline index.cjsx:28]: @pandafulmanda heads up, I modified the exports to be inline.  Happy to revert if you disagree and like the original look.

Comment (pandafulmanda) [inline index.cjsx:28]: hmmmmmmmmm.....i see why, ooooh kay. yah. that's koo

**Comment by nathanstitt** [comment]
Removed blocked label now that all BE PR's are merged

**Pull request merged**